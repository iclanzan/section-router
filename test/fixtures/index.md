# This is the home page

Isnâ€™t this beautiful?
