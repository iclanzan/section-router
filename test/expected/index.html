<p>Isnâ€™t this beautiful?</p>
